### PR TITLE
Remove flask-sqlalchemy

### PR DESCRIPTION
**Pull request by jonahkagan**
Removes the flask-sqlalchemy plugin and sets up sqlalchemy directly. There are a few motivations for this:
- Makes it easier to set up Alembic for db migrations (which we'll need very shortly), which integrates directly with sqlalchemy
- We get typechecking of our db models (which are used in basically all the server code)
- Removes block for @umbernhard to parallelize the tests

This PR includes the following changes:
- Removing the plugin
- Reconfiguring the models to use sqlalchemy directly
- Resolving resulting type errors
- Resolving some lint errors that came up

### CODE REVIEW

File: server/api/audit_boards.py [hunk 1]
```diff
@@ -249,15 +250,19 @@ def set_audit_board_members(
...
-    db.session.commit()
+    db_session.commit()
 
     return jsonify(status="ok")
 
 
 def calculate_risk_measurements(election: Election, round: Round):
+    if not election.risk_limit:  # Shouldn't happen, we need this for typechecking
+        raise Exception("Risk limit not defined")
+    risk_limit: int = election.risk_limit
```
[the remark this answers is not in the document]
Comment (jonahkagan) [reply server/api/audit_boards.py:261]: https://github.com/python/mypy/issues/4245#issuecomment-522101812

**Comment by jonahkagan** [comment]
@benadida ideas on manual testing?

**Comment by jonahkagan** [comment]
I ran through a whole audit on the review app, so that feels like a good starting point for a manual test.

**Comment by benadida** [comment]
> I ran through a whole audit on the review app, so that feels like a good starting point for a manual test.

That is exactly what I meant by "manual testing" so sounds great :)

**Pull request merged**